### ISSUE TITLE
Do we need to normalize the image range to [0,1]?

### ISSUE THREAD

**Issue by John1231983** [kwotsin/transfer_learning_tutorial#14]
Thanks for your great project. It is very useful for me. I would like to ask you two questions about the project:

1. Do we need to normalize the input image to [0,1] (or zero mean, unit variance) as in the pre-processing step? I checked your project and it did not do it

2. In the evaluation phase, why we need to run so much epoch. I think we just take the last checkpoint and feed all images in the validation set, then compute the accuracy.

3. Could you provide a simple inference code that takes an image as input and show the prediction like you shows in the evaluation code, but it just takes image path as input?

Great job

**Comment by kwotsin** [comment]
As with your comments in the other thread, I think the question of "need" really depends on the task you're trying to achieve. IIRC normalizing the inputs is a common thing to do, but a similar effect is also achieved through batch norm in the model, in which case batch norm normalizes the input to each layer that uses it. so your mileage may vary depending on your specific data set. I would say try both ways and test them empirically to see if it works for your use case.

I believe earlier on I was trying to see if the accuracy varies over each epoch based on the earlier version of TF-slim, but if you see that it doesn't you can just change it to 1.

Well instead of using just one image filename, I think you're better off trying to use a list of image filenames, convert it into a tensor, and then read it.
e.g.

```
        #Load the files into one input queue
        images = tf.convert_to_tensor(image_files)
        labels  = tf.convert_to_tensor(labels_list)
        input_queue = tf.train.slice_input_producer([images, labels])

        #Decode the image and annotation raw content
        image = tf.read_file(input_queue[0])
        image = tf.image.decode_image(image, channels=3)
```
